lib/paper_trail/frameworks/rails/engine.rb

```diff
@@ -2,6 +2,11 @@ module PaperTrail
   module Rails
     class Engine < ::Rails::Engine
       paths['app/models'] << 'lib/paper_trail/frameworks/active_record/models'
+      config.paper_trail = ActiveSupport::OrderedOptions.new
```

Comment:
Can you explain how this works? I never understood what is supposed to go in the `Rails::Engine` beyond code for autoloading libraries, this looks neat but I don't understand it